docs/ja/reference/cli.md

```
@@ -92,9 +93,9 @@ geonic entities get urn:ngsi-ld:Room:001
 
 ---
 
-## グローバル オプション
+## グローバルオプション
 
-すべてのコマンドで使用可能です。優先順位のルールについては、[オプション解決順序](#option-resolution-order)を参照してください。
+すべてのコマンドで使用可能です。優先順位のルールについては [オプション解決順序](#option-resolution-order) を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)